### ISSUE TITLE
Return await is unnecessary in the examples

### ISSUE THREAD

**Issue by vassbence** [zeit/swr#215]
Most of the examples have a small error  in the fetch.js files:

```
export default async function (...args) {
  const res = await fetch(...args)
  return await res.json()
}
````

You should just `return res.json()` since the async function implicitly returns a Promise.

[Read more about it here.](https://github.com/eslint/eslint/blob/master/docs/rules/no-return-await.md)

**Comment by vassbence** [comment]
Fixed in the #216 PR.

**Comment by sergiodxa** [comment]
You could also remove the async and do `return fetch(...args).then(res => res.json())` to completely remove the async function, in bundles for old browsers it would give you smaller bundles too.
